npm i -g minecat 报错

```
 npm i -g minecat
npm ERR! code 1
npm ERR! path /Users/linchen/.nvm/versions/node/v20.10.0/lib/node_modules/minecat
npm ERR! command failed
npm ERR! command sh -c node postinstall.js
npm ERR! node:internal/modules/cjs/loader:1147
npm ERR!   throw err;
npm ERR!   ^
npm ERR!
npm ERR! Error: Cannot find module '/Users/linchen/.nvm/versions/node/v20.10.0/lib/node_modules/minecat/postinstall.js'
npm ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:985:27)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
npm ERR!     at node:internal/main/run_main_module:28:49 {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: []
npm ERR! }
npm ERR!
npm ERR! Node.js v20.10.0

npm ERR! A complete log of this run can be found in: /Users/linchen/.npm/_logs/2024-02-28T10_18_48_203Z-debug-0.log
```

--- a/packages/minecat/package.json
+++ b/packages/minecat/package.json
@@ -15,7 +15,8 @@
   "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "postinstall.js"
   ],
   "scripts": {
     "build": "tsup src -- --dts-resolve",

